1.6.0 broke selinux again


BUG REPORT
Versions
kubeadm version (use kubeadm version):
kubeadm version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.0", GitCommit:"fff5156092b56e6bd60fff75aad4dc9de6b6ef37", GitTreeState:"clean", BuildDate:"2017-03-28T16:24:30Z", GoVersion:"go1.7.5", Compiler:"gc", Platform:"linux/amd64"}
Environment:

Kubernetes version (use kubectl version):

Client Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.0", GitCommit:"fff5156092b56e6bd60fff75aad4dc9de6b6ef37", GitTreeState:"clean", BuildDate:"2017-03-28T16:36:33Z", GoVersion:"go1.7.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.0", GitCommit:"fff5156092b56e6bd60fff75aad4dc9de6b6ef37", GitTreeState:"clean", BuildDate:"2017-03-28T16:24:30Z", GoVersion:"go1.7.5", Compiler:"gc", Platform:"linux/amd64"}


What happened?
On the master, where you run kubeadm init, setenforce 0 is still needed as it will be denied to read the certs.
 ausearch -m avc
type=AVC msg=audit(1490892442.272:1130): avc:  denied  { open } for  pid=5137 comm="kube-controller" path="/etc/kubernetes/pki/sa.key" dev="sda1" ino=1573694 scontext=system_u:system_r:container_t:s0:c483,c527 tcontext=unconfined_u:object_r:cert_t:s0 tclass=file permissive=1
----
time->Thu Mar 30 16:47:49 2017
type=AVC msg=audit(1490892469.704:1170): avc:  denied  { read } for  pid=7452 comm="kube-apiserver" name="apiserver.crt" dev="sda1" ino=1573691 scontext=system_u:system_r:container_t:s0:c11,c526 tcontext=unconfined_u:object_r:cert_t:s0 tclass=file permissive=1
----
time->Thu Mar 30 16:47:49 2017
type=AVC msg=audit(1490892469.704:1171): avc:  denied  { open } for  pid=7452 comm="kube-apiserver" path="/etc/kubernetes/pki/apiserver.crt" dev="sda1" ino=1573691 scontext=system_u:system_r:container_t:s0:c11,c526 tcontext=unconfined_u:object_r:cert_t:s0 tclass=file permissive=1
----
time->Thu Mar 30 16:47:52 2017
type=AVC msg=audit(1490892472.814:1173): avc:  denied  { open } for  pid=7326 comm="kube-controller" path="/etc/kubernetes/pki/sa.key" dev="sda1" ino=1573694 scontext=system_u:system_r:container_t:s0:c391,c896 tcontext=unconfined_u:object_r:cert_t:s0 tclass=file permissive=1
----
time->Thu Mar 30 16:47:52 2017
type=AVC msg=audit(1490892472.814:1172): avc:  denied  { read } for  pid=7326 comm="kube-controller" name="sa.key" dev="sda1" ino=1573694 scontext=system_u:system_r:container_t:s0:c391,c896 tcontext=unconfined_u:object_r:cert_t:s0 tclass=file permissive=1

What you expected to happen?
Seems similar to #107, and expected to run with setenforce 1
How to reproduce it (as minimally and precisely as possible)?
Every time
Anything else we need to know?
